src/main/java/com/laserfiche/repository/api/clients/params/ParametersForAssignFieldValues.java

```diff
@@ -4,6 +4,9 @@
 
 import java.util.Map;
 
+/**
+ * The encapsulated parameters for {@link com.laserfiche.repository.api.clients.impl.EntriesClientImpl#assignFieldValues(ParametersForAssignFieldValues) assignFieldValues}.
```

Comment:
This is not clear for me.
Shouldn't it be just 
{@link com.laserfiche.repository.api.clients.impl.EntriesClientImpl#assignFieldValues(ParametersForAssignFieldValues)}

instead of 
{@link com.laserfiche.repository.api.clients.impl.EntriesClientImpl#assignFieldValues(ParametersForAssignFieldValues) assignFieldValues}

Reply:
If the javadoc was `{@link com.laserfiche.repository.api.clients.impl.EntriesClientImpl#assignFieldValues(ParametersForAssignFieldValues)}` then it would look like this in the ide.
![image](https://user-images.githubusercontent.com/102689343/211060147-b86db77f-44d0-4140-8394-08753541ce5e.png)

If the javadoc was `{@link com.laserfiche.repository.api.clients.impl.EntriesClientImpl#assignFieldValues(ParametersForAssignFieldValues) assignFieldValues}` then it would look like this in the ide.
![image](https://user-images.githubusercontent.com/102689343/211060256-b6d1de96-4b23-4a95-840f-9c58ab010a7d.png)

The second parameter in the `@link` is a text label to make the documentation less bloated with the namespace information. But if you think that information is good, then I can use the first option instead. Or maybe I can change the text label to `EntriesClientImpl.assignFieldValues` if you think that is better

Reply:
Yes. You are correct. The second screenshot is better, as it is less verbose. Also the assignFieldValues is clickable, so, don't think it needs to be the fully-qualified name.
So, the current code is ok.